src/handler/get_address.c

```diff
@@ -14,18 +14,29 @@ void derive_address() {
     cx_ecfp_private_key_t private_key = {0};
     cx_ecfp_public_key_t public_key = {0};
     uint8_t chain_code[32];
+    uint32_t error = CX_OK;
```

Comment:
This is required by `CX_CHECK`, since this macro sets the value of `error` to the return of the expression and jumps to the end if it's not `CX_OK`.